docs/pages/connect-your-client/vnet.mdx

```diff
@@ -0,0 +1,93 @@
+`tcp-app.teleport.example.com` instead of talking to a port on localhost. Underneath, VNet uses [the
+same TLS routing technology](../architecture/tls-routing.mdx) as `tsh proxy app`, just with extra
+DNS and virtual network setup on the client side.
+
+## Prerequisites
+
+- A client machine running macOS Ventura (13.0) or higher.
+- [Teleport Connect](teleport-connect.mdx), version 16.0.0 or higher.
+
+## Step 1/3. Start Teleport Connect
+
+Open Teleport Connect and log in to the cluster. Find the TCP app you want to connect to. TCP apps
+have `tcp://` as the protocol in their addresses.
+
+![Resource list in Teleport Connect with a TCP hovered over](../../img/use-teleport/vnet-resources-list@2x.png)
+
+## Step 2/3. Start VNet
+
+Click "Connect" next to the TCP app. This starts VNet if it's not already running. Alternatively,
+you can start VNet through the connection list in the top left.
+
+VNet needs to set up a network device and configure DNS on your device, so it'll ask for admin
+permissions. For now your password is needed each time you start VNet, but we're working on showing
+the prompt only once.
+
+![VNet starting up](../../img/use-teleport/vnet-starting@2x.png)
+
+## Step 3/3. Connect
+
+Once VNet is running, you can connect to the application using the application client you would
+normally use to connect to it. The port number is inconsequential. VNet forwards connections over
+any port number under the public address of the app to the app itself.
+
+```code
+$ psql postgres://postgres@tcp-app.teleport.example.com/postgres
+```
+
+VNet is going to automatically start on the next Teleport Connect launch, unless you stop VNet
+before closing Teleport Connect.
+
+## `tsh` support
+
+VNet is available in `tsh` as well. Using it is as simple as logging in to the cluster and
+calling `tsh vnet`.
+
+```code
+$ tsh login --proxy=teleport.example.com
+$ tsh vnet
+```
+
+## Troubleshooting
+
+### Timeouts when trying to reach a Teleport cluster
+
+If VNet doesn't have a chance to clean up before stopping, such as during sudden device shut down,
+it may leave leftover DNS configuration files in `/etc/resolver`. Those files tell your computer to
+talk to a DNS server operated by VNet when connecting to your cluster. But since VNet is no longer
+running, there's no DNS server to answer those calls.
+
+To clean up those files, simply start VNet again. Alternatively, you can remove the leftover files
+manually.
+
+### Submitting an issue
+
+When [submitting an
+issue](https://github.com/gravitational/teleport/issues/new?assignees=&labels=bug,vnet&template=bug_report.md),
+make sure to include logs from `/var/log/vnet.log`, as well as [Teleport Connect
+logs](teleport-connect.mdx#submitting-an-issue) or the output of `tsh vnet -d`, depending on which
+client you use.
+
+## Next steps
+
+- Read [RFD 163](https://github.com/gravitational/teleport/blob/master/rfd/0163-vnet.md) to learn how VNet works on a technical level.
```

Comment:
Do we ever need a proper explanation behind VNet? Would we put it in admin docs or somewhere else?

I imagine some sort of simplified explanation of the DNS setup would be useful.